src/test/java/org/simulator/sbml/SBMLTestSuiteRunner.java

```diff
@@ -969,7 +969,7 @@ public void testModels() throws FileNotFoundException, IOException {
             boolean errorInSolve = false;
             try {
               solution = solver.solve(interpreter,
-                interpreter.getInitialValues(), timepoints);
+                interpreter.getInitialValues(), timepoints, null);
```

Comment:
Please remove the tab.

Please also note the same situation in other objects.

Reply:
I am not understanding what are you meaning by tab?